.env.example

```
@@ -110,6 +110,9 @@ SEND_MAILS_TAGS=dev,localhost
 # Connection URL to e.g. SMTP account nodemailer uses.
 #SEND_MAILS_NODEMAILER_CONNECTION_URL='smtps://inbox@domain.tld:lengthy-password-with-dashes@asmtp.mail.hostpoint.ch/?pool=true'
 
+# If Regular Expression is matched on a template name, mail is schedule instead of sent
+#SEND_MAILS_SCHEDULE_REGEX=^[^signin].*
```

```suggestion
#SEND_MAILS_SCHEDULE_TEMPLATE_REGEX=^[^signin].*
```
to distinguish it from email-address regex. applies to `SEND_MAILS_NODEMAILER_REGEX` too, sorry I missed it there.

---

Yeah, true. This env variable names keep growing and growing and growing.